src/wasm-traversal.h

```diff
@@ -473,7 +473,7 @@ struct Walker : public VisitorType {
   static void doVisitAtomicRMW(SubType* self, Expression** currp)    { self->visitAtomicRMW((*currp)->cast<AtomicRMW>()); }
   static void doVisitAtomicCmpxchg(SubType* self, Expression** currp){ self->visitAtomicCmpxchg((*currp)->cast<AtomicCmpxchg>()); }
   static void doVisitAtomicWait(SubType* self, Expression** currp)   { self->visitAtomicWait((*currp)->cast<AtomicWait>()); }
-  static void doVisitAtomicWake(SubType* self, Expression** currp)   { self->visitAtomicWake((*currp)->cast<AtomicWake>()); }
+  static void doVisitAtomicNotify(SubType* self, Expression** currp)   { self->visitAtomicNotify((*currp)->cast<AtomicNotify>()); }
```

Comment:
indentation of the `{` compared to other lines should stay consistent as beore

Reply:
Done